src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,57 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+
+    // if (typeof text !== "string") {
+    //   return "Please enter a valid string";
+    // }
```

Comment:
This is the recommended approach, why did you comment it out? Although, what you need to do is `throw Error` rather than `return ...`

Reply:
Thanks, noted